bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/launch/LaunchUtils.java

```diff
@@ -94,13 +93,30 @@ public static void stopListeners(boolean isService) {
         listeners.forEach(listener -> listener.afterRunProgram(isService));
     }
 
+    private static VariableKey[] mergeKeys(VariableKey[] keyArray1, VariableKey[] keyArray2) {
+
+        if (keyArray1 == null && keyArray2 == null) {
```

Comment:
We do not need to check null here since they will never be null.

Reply:
Got it. I'll remove that check.

Reply:
I'll remove the check.